slime/backends/fsdp_utils/arguments.py

```diff
@@ -26,6 +26,10 @@ class FSDPArgs:
     fsdp_sync_module_states: bool = True
     fsdp_forward_prefetch: bool = True
     fsdp_backward_prefetch: bool = True
+    attn_implementation: str = "flash_attention_2"
+
+    # max_tokens_per_gpu is defined in main arguments (slime/utils/arguments.py)
+    micro_batch_size: int = 1  # Micro batch size
```

Comment:
I think that this arg should already been registered in `slime/utils/arguments.py`?